lib/element_mapping.js

```diff
@@ -292,7 +296,12 @@ module.exports = {
     }
   },
   'core-tooltip': {
-    noReplacement: true
+    name: 'paper-tooltip',
+    notice: 'Whereas core-tooltip was intended to contain the element it ' +
```

Comment:
Make this notice more ominous? It'll probably add the tooltip to the wrong element.